src/app/client/src/app/modules/resource/modules/player/components/collection-player/collection-player.component.ts

```diff
@@ -524,6 +525,7 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy, AfterViewIn
   clearSelection() {
     this.isCopyAsCourseClicked = !this.isCopyAsCourseClicked;
     this.selectAll = false;
+    this.enableCreateButton = false;
```

Comment:
Replace this with selectedItems array object.

Reply:
done